surround-lib/README.md

```diff
@@ -0,0 +1,126 @@
+<p align="center">
+    <img src="./docs/source/temp_logo_hq.png" width="500">
```

Comment:
Same as the CLI readme. This needs to be  split into the appropriate place of each

Reply:
Same here